### PR TITLE
feat: proper Atlas revert page

### PR DESCRIPTION
**Pull request by he3als**
### Type
- [x] Rewrite already written documentation
- [x] Add/remove new pages

### Questions
- [x] Did you preview your changes beforehand?
- [x] Did you read the [Atlas Contribution Guidelines](https://docs.atlasos.net/contributions/)?

### Describe your pull request
Adds a proper revert page that uses snippets from the main install page. The actual Markdown file needs to be in `getting-started` due to how snippets work with relative links.

https://revert-atlas-page.atlasos-docs-enq.pages.dev/getting-started/reverting-atlas/

### CODE REVIEW

File: docs/getting-started/reverting-atlas.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+This page details how to reinstall from Atlas to stock Windows fully, which uninstalls Atlas.
+
+## :material-handshake: Before uninstalling
+
+If you're experiencing issues, ensure you're on the [latest version of Atlas](installation.md), and then consider getting help from the following:
+
+- :material-file-search: The documentation you're on currently
+- <a onclick="clickKapaAi();" href="javascript:void(0)">
+    <p>:material-robot: 'Ask AI' in the corner of this page</p>
```
Comment (the-P1neapple) [inline docs/getting-started/reverting-atlas.md:19]: paragraph tag is causing formatting issue

**Review by the-P1neapple — CHANGES_REQUESTED** [review]

File: docs/getting-started/reverting-atlas.md (hunk 1 above)
Comment (he3als) [reply docs/getting-started/reverting-atlas.md:19]: https://github.com/Atlas-OS/docs/assets/65787561/7f749817-09a9-4c23-ae5d-dba89d4ce563

Where?

Comment (the-P1neapple) [reply docs/getting-started/reverting-atlas.md:19]: As shown on desktop
image

Comment (he3als) [reply docs/getting-started/reverting-atlas.md:19]: Which browser?

Comment (the-P1neapple) [reply docs/getting-started/reverting-atlas.md:19]: Currently using firefox developer edition (latest version)

**Review by the-P1neapple — APPROVED** [review]

**Review by jacksyrm — APPROVED** [review]

**Review by the-P1neapple — APPROVED** [review]

**Review by whallin — APPROVED** [review]

**Pull request merged**